starwars/src/App.js

```diff
@@ -1,19 +1,29 @@
-import React from 'react';
-import './App.css';
+import React, { useState, useEffect } from "react";
+import "./App.css";
+import axios from "axios";
+import CharCard from "./components/characterCards";
 
 const App = () => {
-  // Try to think through what state you'll need for this app before starting. Then build out
-  // the state properties here.
-
-  // Fetch characters from the star wars api in an effect hook. Remember, anytime you have a 
-  // side effect in a component, you want to think about which state and/or props it should
-  // sync up with, if any.
-
+  const [CharacterData, setCharacterData] = useState([]);
```

Comment:
Great job creating the variable and function of useState. My only recommendation would be to only use capitalized variables for Components.